dev-python/pyside2/metadata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>leycec@gmail.com</email>
```

Comment:
Who is this?

Reply:
@leycec from the raiagent overlay where I found this ebuild, I have slightly edited the ebuild. But I seem to have forgotten to add myself to the metadata file, I fixed that now. I left leycec in there because I figured it would be rude to just remove them from the metadata file.